messenger.rst

```diff
@@ -730,6 +730,14 @@ options.
 AMQP Transport
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 5.1
+
+    The AMQP transport has moved to a separate package. Install it by running
+
+    .. code-block:: terminal
```

Comment:
The rendering of a "code-block" inside a "versionadded" is not very well supported.
I have this rendering : 
![Capture d’écran 2020-03-04 à 17 36 19](https://user-images.githubusercontent.com/12966574/75901416-bff96f80-5e3e-11ea-8372-06b80678944c.png)

Reply:
It would be better to have something like this : 
![Capture d’écran 2020-03-04 à 17 39 34](https://user-images.githubusercontent.com/12966574/75901926-8d9c4200-5e3f-11ea-9dda-f8d13a59587e.png)